src/components/UserInfo.js

```diff
@@ -1,21 +1,31 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, jobSelector, avatorselector) {
     this._name = document.querySelector(nameSelector);
     this._job = document.querySelector(jobSelector);
+    this._avatar = document.querySelector(avatorselector);
   }
 
   //   returns an object with information about the user to display the user data in the open form.
   getUserInfo() {
     this._userInfo = {
       name: this._name.textContent,
-      description: this._job.textContent,
+      about: this._job.textContent,
     };
     return this._userInfo;
   }
 
+  getUserPhoto() {
+    this._userInfo = { avatar: this._avatar.src };
```

Comment:
Needs correction: You should not store variables in class instance `this` until it's needed. If the variables are not used in other methods, then it’s needed to declare them as local constants.

```js
const userInfo = ...
```

The same for `  getUserInfo` method